mne_bids/utils.py

```diff
@@ -76,11 +76,13 @@ def _get_ch_type_mapping(fro='mne', to='bids'):
 
     elif fro == 'bids' and to == 'mne':
         mapping = dict(EEG='eeg', MISC='misc', TRIG='stim', EMG='emg',
-                       ECOG='ecog', SEEG='seeg', EOG='eog', ECG='ecg',
+                       ECOG='ecog', SEEG='seeg', ECG='ecg',
                        RESP='resp', NIRS='fnirs_cw_amplitude',
+                       DBS='dbs',
                        # No MEG channels for now
-                       # Many to one mapping
-                       VEOG='eog', HEOG='eog', DBS='dbs')
+                       # Many-to-one mappings
+                       EOG='eog', VEOG='eog', HEOG='eog',
+                       TEMP='misc', GSR='misc',)
```

Comment:
to me TEMP and GSR are not part of the standard

this data contains channels with type "temp" and "GSR"

Cf https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/03-electroencephalography.html#channels-description-_channelstsv

Type of channel; MUST use the channel types listed below. Note that the type MUST be in upper-case.  

Must be one of: "MEGMAG", "MEGGRADAXIAL", "MEGGRADPLANAR", "MEGREFMAG", "MEGREFGRADAXIAL", "MEGREFGRADPLANAR", "MEGOTHER", "EEG", "ECOG", "SEEG", "DBS", "VEOG", "HEOG", "EOG", "ECG", "EMG", "TRIG", "AUDIO", "PD", "EYEGAZE", "PUPIL", "MISC", "SYSCLOCK", "ADC", "DAC", "HLU", "FITERR", "OTHER".

Reply:
The list is outdated. Scroll down a little more to see a table with all supported types. The list should be removed and instead a reference to the table should be added.